rules/windows/defense_evasion_clearing_windows_console_history.toml

```
@@ -0,0 +1,52 @@
+[metadata]
+creation_date = "2021/11/22"
+maturity = "production"
+updated_date = "2021/11/22"
+
+[rule]
+author = ["Austin Songer"]
+description = """
+Identifies when an user attempts to clear console history. An adversary may clear the command history of a compromised account to conceal
+the actions undertaken during an intrusion.
+"""
+from = "now-9m"
+index = ["winlogbeat-*", "logs-endpoint.events.*", "logs-windows.*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Clearing Windows Console History"
+references = [
+    "https://stefanos.cloud/blog/kb/how-to-clear-the-powershell-command-history/",
+    "https://www.shellhacks.com/clear-history-powershell/",
+    "https://community.sophos.com/sophos-labs/b/blog/posts/powershell-command-history-forensics",
+]
+risk_score = 21
+rule_id = "b5877334-677f-4fb9-86d5-a9721274223b"
+severity = "low"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Defense Evasion"]
+timestamp_override = "event.ingested"
+type = "eql"
+
+query = '''
+process where event.type in ("process_started", "start") and
+   (process.name : ("pwsh.exe", "powershell.exe", "powershell_ise.exe") or process.pe.original_file_name in
+     ("pwsh.exe", "powershell.exe", "powershell_ise.exe") and
```

```suggestion
     ("pwsh.dll", "powershell.exe", "powershell_ise.exe") and
```
As we saw last time, the original name for pwsh.exe is pwsh.dll